index.bs

```diff
@@ -80,8 +80,18 @@ and occurs frequently on the web today. This shifting is often due to content
 loading asynchronously and displacing other elements on the page.
 
 The layout Instability API identifies these unstable pages by reporting a value
-(the "layout shift") for each animation frame in the user's session, allowing
-the developer to compute a cumulative layout shift score for the page.
+(the "layout shift") for each animation frame in the user's session.  This
```

Comment:
Perhaps we can describe this as proportional to the area of shift multiplied and also to the distance shifted?

Reply:
Updated, PTAL